src/pages/components/content-switcher/code.mdx

```diff
@@ -4,34 +4,180 @@ description: Content switchers manipulate the content shown following an exclusi
-  <Switch name={'second'} text='Second section' />
-  <Switch name={'third'} text='Third section' />
+<PageDescription>
+
+How to build a content switcher using React. For code usage with other frameworks, please follow the links in the live demo on the usage tab.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Component API</AnchorLink>
+<AnchorLink>Accessibility considerations</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+You can build content switcher by using a combination of the `ContentSwitcher` and `Switch` components. The `ContentSwitcher` components accepts a list of `Switch` components as children and each `Switch` is responsible for displaying the section heading.
+
+You can configure the `Switch` heading using the `text` prop.
+
+```jsx
+import { ContentSwitcher, Switch } from "carbon-components-react";
+
+function MyComponent() {
+  return (
+    <ContentSwitcher onChange={function noRefCheck() {}}>
+      <Switch
+        onClick={function noRefCheck() {}}
+        onKeyDown={function noRefCheck() {}}
+        text="First section"
+      />
+      <Switch
+        onClick={function noRefCheck() {}}
+        onKeyDown={function noRefCheck() {}}
+        text="Second section"
+      />
+      <Switch
+        onClick={function noRefCheck() {}}
+        onKeyDown={function noRefCheck() {}}
+        text="Third section"
+      />
+    </ContentSwitcher>
+  );
+}
+```
+
+## Component API
+
+### ContentSwitcher props
+
+| Prop          | Type                         | Required | Default     | Description                                                                                              |
+| ------------- | ---------------------------- | -------- | ----------- | -------------------------------------------------------------------------------------------------------- |
+| selectedIndex | `number`                     | –        | 0           | Specify a selected index for the initially selected content                                              |
+| selectionMode | oneOf 'automatic' / 'manual' | –        | 'automatic' | Choose whether or not to automatically change selection on focus                                         |
+| children      | `node`                       | `true`   | –           | Pass in Switch components to be rendered in the ContentSwitcher                                          |
+| className     | `string`                     | –        | –           | Specify an optional className to be added to the container node                                          |
+| onChange      | `func`                       | 'true'   | –           | Specify an `onChange` handler that is called whenever the ContentSwitcher changes which item is selected |
+
+### ContentSwitcher selectedIndex
+
+You can render the content switcher with a different section automatically selected by using `selectedIndex`.
```

Comment:
```suggestion
You can control which item is selected for the content switcher by using the `selectedIndex` prop.
```

Might help to use "control" as the terminology since this is a controlled component case.